user_workspaces_server/tasks.py

```diff
@@ -109,14 +115,41 @@ def delete_workspace(workspace_id):
         async_task('user_workspaces_server.tasks.update_user_quota_disk_space', user_quota.id)
 
 
-def update_workspace_disk_space(workspace_id):
+def update_workspace(workspace_id):
     try:
         workspace = models.Workspace.objects.get(pk=workspace_id)
     except Exception as e:
         print(repr(e))
         raise e
 
     main_storage = apps.get_app_config('user_workspaces_server').main_storage
+    current_details = {
+        "files": [],
+        "symlinks": []
+    }
+
+    # This will IGNORE dot directories and files
+    for dirpath, dirnames, filenames, dirfd in main_storage.get_dir_tree(workspace.file_path):
+        dirnames[:] = [dirname for dirname in dirnames if not dirname[0] == '.']
+        filenames = [f for f in filenames if not f[0] == '.']
```

Comment:
Why the `[:]`? Are you particularly concerned about not creating a new array?

Reply:
Using [:] makes it so that you do not go **into** hidden directories.